cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,24 @@
-/// <reference types="cypress" />
+/// <reference types="cypress" />;
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('/user/login');
+  });
+
   it('should provide an ability to log in', () => {
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+
+    cy.get('[placeholder=Email]')
+      .type('melamori1605@gmail.com');
+
+    cy.get('[placeholder=Password]')
+      .type('123456Test!!');
+
+    cy.get('.btn[type=submit]')
+      .click();
 
+    cy.get('.navbar-nav li:last-child .nav-link')
+      .type('calculatingmodel');
```

Comment:
It seems like there is an error here. The `.type('calculatingmodel')` method is being used on a navigation link, which is not an input field. You might want to use an assertion to check if the user is logged in, such as verifying the presence of a username or a logout button.